docs/en/operations/settings/settings.md

```diff
@@ -2459,4 +2458,65 @@ Possible values:
 
 Default value: `0`.
 
+## union_default_mode {#union-default-mode}
+
+Sets a special mode for combining `SELECT` query results using the [UNION](../../sql-reference/statements/select/union-all.md) expression.
```

Comment:
It should be highlighted that it relates only to UNION without explicit UNION ALL or UNION DISTINCT specification.